dll/win32/msgina/gui.c

```diff
@@ -30,6 +32,7 @@ typedef struct _LEGALNOTICEDATA
 
 // Timer ID for the animated dialog bar.
 #define IDT_BAR 1
+#define ISKEYDOWN(x) GetKeyState(x) & 0x8000
```

Comment:
Not still sure of the utility of this macro.
Also its definition should be parenthesed:
```suggestion

#define ISKEYDOWN(x) (GetKeyState(x) & 0x8000)
```

Reply:
This macro makes it easier to understand what our code is looking for instead of using some bitwise magic every time we want to check what key is being held down.

Reply:
The macro could possibly be omitted if you replace the magic value with a defined constant (but I'm not sure we have definition for it).